chain/consensus/mir/config.go

```diff
@@ -56,13 +60,25 @@ type Config struct {
 	Consensus *ConsensusConfig
 }
 
+func DefaultConsensusConfig() *ConsensusConfig {
+	return &ConsensusConfig{
+		SegmentLength:                DefaultSegmentLength,
+		ConfigOffset:                 DefaultConfigOffset,
+		MaxTransactionsInBatch:       DefaultMaxTransactionsInBatch,
+		MaxProposeDelay:              DefaultMaxBlockDelay,
+		PBFTViewChangeSNTimeout:      DefaultPBFTViewChangeSNTimeout,
+		PBFTViewChangeSegmentTimeout: DefaultPBFTViewChangeSegmentTimeout,
+	}
+
+}
+
 func NewConfig(
 	addr address.Address,
 	dbPath string,
 	initCheck *checkpoint.StableCheckpoint,
 	checkpointRepo string,
 	segmentLength, configOffset int,
-	maxBlockDelay time.Duration,
+	maxBlockDelay int,
```

Comment:
Why did this change to int? It would make more sense to me if it was `time.Duration`